### PR TITLE
[BH-2005] Add volume setting for focus timer

### PR DESCRIPTION
**Pull request by Maciej-Mudita**
The user will be able to control the volume of notifications in the Focus Timer application

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [x] Has changelog entry added

### CODE REVIEW

File: image/system_a/data/lang/Polski.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "app_bell_focus_long_break_after": "D\u0142uga przerwa po",
     "app_bell_focus_time_once": "czasie skupienia",
     "app_bell_focus_time_many": "czasach skupienia",
+    "app_bell_focus_notification_volume": "Głośność powiadomień",
```
Comment (dawidwojtas-mudita) [inline image/system_a/data/lang/Polski.json:43]: What about Unicode characters like `ł` -> `\u0142`?

File: products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusSettingsPresenter.cpp [hunk 2]
```diff
@@ -5,15 +5,28 @@
...
+#include <common/widgets/list_items/NumericWithBar.hpp>
 #include <common/widgets/list_items/MinutesWithOff.hpp>
 #include <common/widgets/list_items/Multiplicity.hpp>
+#include <gui/widgets/ListViewEngine.hpp>
 
 namespace
 {
+    enum class ListIndex
+    {
+        focusTime = 0,
```
Comment (dawidwojtas-mudita) [inline products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusSettingsPresenter.cpp:22]: What about capital letters for enums? ;)

**Review by dawidwojtas-mudita — APPROVED** [review]

File: image/system_a/data/lang/Polski.json (hunk 1 above)
Comment (Maciej-Mudita) [reply image/system_a/data/lang/Polski.json]: done

File: products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusSettingsPresenter.cpp (hunk 2 above)
Comment (Maciej-Mudita) [reply products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusSettingsPresenter.cpp]: done

**Review by Lefucjusz — APPROVED** [review]

**Pull request merged**